index.bs

```diff
@@ -7914,10 +7904,25 @@ Constructors</h4>
 				[[mediacapture-streams#mediastream|MediaStream]] that has at least one
 				[[mediacapture-streams#mediastreamtrack|MediaStramTrack]] whose
 				<code>kind</code> attribute has the value <code>"audio"</code>,
-				throw an {{InvalidStateError}} and abort these steps.
-			3. Let <var>node</var> be a new {{MediaStreamAudioSourceNode}}
+				throw an {{InvalidStateError}} and abort these steps. Else, let
+				this stream be <var>inputStream</var>.
+			3. Let <var>tracks</var> the list of all
+				[[mediacapture-streams#mediastreamtrack|MediaStreamTrack]] of
+				<var>inputStream</var> that have a <code>kind</code> of
+				<code>"audio"</code>.
+			4. Sort the elements in <var>tracks</var> based on their <code>id</code>
+				attribute, ordering by Unicode code point
+			5. Let the first element of <var>tracks</var> be the
+				[[mediacapture-streams#mediastreamtrack|MediaStreamTrack]] used as the
+				input audio for this {{MediaStreamAudioSourceNode}}.
+			6. Let <var>node</var> be a new {{MediaStreamAudioSourceNode}}
```

Comment:
Also how do we attach these "sorted tracks" to the node? Should it be explicit? or is it fine as it is now?

Reply:
I was thinking maybe that we should stick the chosen track into a private slot, what do you think?

Reply:
Specifically, the other tracks aren't mentioned, because regardless of what happens to them, it does not affect this node. I think it's still good to stick the track into a slot though.

In general, those `Media*AudioSourceNode` are a bit hand-wavy when it comes to the actual processing.

It's not that hard, but implementations vary widely (in terms of latency, notably), and, for example what happens when the input CORS-status changes during processing is a bit unclear.

Reply:
+1 on putting it into a private slot. Clarifying behaviors of `Media*AudioSourceNode` can be a good V2 issue.